ios/RCCDrawerController/RCCDrawerController.m

```diff
@@ -13,25 +13,28 @@ @implementation RCCDrawerController
 
 @synthesize drawerStyle = _drawerStyle;
 
+UIViewController *leftViewController = nil;
+UIViewController *rightViewController = nil;
+
 -(UIInterfaceOrientationMask)supportedInterfaceOrientations {
     return [self supportedControllerOrientations];
 }
 
 - (instancetype)initWithProps:(NSDictionary *)props children:(NSArray *)children globalProps:(NSDictionary*)globalProps bridge:(RCTBridge *)bridge
 {
-    
+
     self.drawerStyle = props[@"style"];
-    
+
     // center
     if ([children count] < 1) return nil;
     UIViewController *centerViewController = [RCCViewController controllerWithLayout:children[0] globalProps:globalProps bridge:bridge];
-    
+
     // left
     UIViewController *leftViewController = nil;
```

Comment:
Because leftViewController and rightViewController are defined locally here, the class variables never get set. When you go to actually set the status of one of the drawers, the controller is therefore always nil (that caused me quite some confusion, as MMDrawerController doesn't handle that situation nicely and permanently disables interaction on the UI).